helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -238,7 +239,6 @@ public ZKHelixManager(String clusterName, String instanceName, InstanceType inst
       HelixManagerProperty helixManagerProperty) {
     validateZkConnectionSettings(zkAddress, helixManagerProperty);
 
-    _zkAddress = zkAddress;
```

Comment:
any particular reason, zkAddress is not set here. Customer has provided as input.